.travis.yml

```diff
@@ -1,7 +1,26 @@
 language: python
+sudo: false
 python:
   - '2.7_with_system_site_packages'
   - 2.6
+addons:
+  apt:
+    packages:
+    - python-qt4
+    - python-qt4-dev
+    - python-qt4-gl
+    - python-pip
+    - python-numpy
+    - python-wxtools
+    - ccache
+    - cmake
+    - swig
+    - zlib1g-dev
+    - libpng-dev
+    - libfreetype6-dev
+    - libcairo2-dev
```

Comment:
We should also not need to install this, we should let `python-cairo` bring any dependencies it needs.